plugins/outputs/kinesis/kinesis.go

```diff
@@ -30,9 +30,10 @@ type (
 		Partition          *Partition `toml:"partition"`
 		Debug              bool       `toml:"debug"`
 
-		Log        telegraf.Logger `toml:"-"`
-		serializer serializers.Serializer
-		svc        kinesisiface.KinesisAPI
+		Log                  telegraf.Logger `toml:"-"`
+		maxRecordsPerRequest uint32
```

Comment:
Adding `maxRecordsPerRequest` to make unit testing easier.

Reply:
Hmmm I think this should be a `const` shouldn't it? It will probably never change at runtime...

Reply:
If it's a `const`, this would require creating 500+ records for testing batching.  While this is doable, it's not really practical.

Reply:
Done